docs/_articles/ja/apex/apex-get-started.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Quick Start Apex Development with Salesforce Extensions
+lang: en
+---
+  
+## Quick Start Apex Development
+Complete this Quick Start to create, test and debug a workflow that is created using an Apex class, and sets off a Apex trigger.
+
+We’ll create a custom object called `Book`, and add a custom field called `Price` to that custom object. We’ll update and test the updated Price field using Apex and learn how to work with Apex Code using the Salesforce Extensions.
+
+
+Before you start, install [Salesforce Extensions for Desktop](./en/../../vscode-desktop/install.md) or [Set Up Code Builder](./en/../../codebuilder/cb-setup.md)
+
+
+### Create a Custom Object
+1. Log in to your sandbox or Developer org.
+2. Go to **Setup** > **Object Manager** > **Create** > **Custom Object**.
+3. Enter `Book` for the label.
+4. Enter `Books` for the plural label.
+5. Click **Save.**
+6. In the **Custom Fields & Relationships** section of the Book detail page, click **New**.
+7. Select `Number` for the data type and click **Next**.
+8. Enter `Price` for the field label.
+9. Enter `16` in the length text box.
+10. Enter `2` in the decimal places text box, and click Next.
+11. Click **Next** to accept the default values for field-level security.
+12. Click **Save**.
+
+### Add an Apex Class
+1. Connect to the org that you added the custom object to in VS Code.
+2. Run the `SFDX: Refresh SObject Definitions` command from the Command Palette to get completion suggestions for your SObjects related code.
+3. Scroll down to Custom Object in the Org Browser and locate the `Book_c` object. Click the retrieve icon to run `SFDX: Retrieve Source from Org`.
+4. From the command palette run `SFDX:Create Apex Class` and create a class called `MyHelloWorld`. 
+5. Add a method called `applyDiscount` to this class. 
+   
+We’ll make this method both public and static. Because it’s a static method, you don't need to create an instance of the class to access the method—you can just use the name of the class followed by a dot (.) and the name of the method.
```

Comment:
```suggestion
Make this method both public and static. Because it’s a static method, you don't need to create an instance of the class to access the method — you can just use the name of the class followed by a dot (.) and the name of the method.
```